### PR TITLE
connected peers

### PR DESCRIPTION
**Pull request by mystical-prog**
## What was wrong?
Missing feature to easily get a list of connected peers

Issue #419 

## How was it fixed?

I have created a new method `get_connected_peers` in `IHost` class which returns a list of peer ids of currently connected peers, I have created a new test file `/tests/core/host/test_connected_peers.py` which performs necessary checks for the new method.

### To-Do

- [X] Ran `make lint`
- [X] Ran tests
- [X] Added `newsfragment`

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

**Comment by mystical-prog** [comment]
@pacrob awaiting your review on this PR

File: tests/core/host/test_connected_peers.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    [
+        (connect_two),
+        (connect_three_cyclic),
+        (connect_two_to_one),
+        (connect_and_disconnect),
+    ],
+)
+@pytest.mark.trio
+async def test_connected_peers(test, security_protocol):
+    print("!@# ", security_protocol)
```
Comment (pacrob) [inline tests/core/host/test_connected_peers.py:110]: Did you mean to live this line and the commented lines below in?

Comment (mystical-prog) [reply tests/core/host/test_connected_peers.py:110]: they exist because i took reference of existing tests, i will get rid of them and push the commit

**Comment by mystical-prog** [comment]
I have pushed another commit, can you please check and let me know @pacrob

**Pull request merged**